rest/mgmt_storage/2019-06-01/src/client.rs

```diff
@@ -0,0 +1,1562 @@
+    }
+    req_builder = req_builder.query(&[("api-version", &configuration.api_version)]);
+    let req = req_builder.build()?;
+    let res = client.execute(req).await?;
+    Ok(res.json().await?)
+}
+pub async fn management_policies_delete(
+    configuration: &Configuration,
+    resource_group_name: &str,
+    account_name: &str,
+    subscription_id: &str,
+    management_policy_name: &str,
+) -> Result<()> {
+    let client = &configuration.client;
+    let uri_str = &format!(
+        "{}/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Storage/storageAccounts/{}/managementPolicies/{}",
+        &configuration.base_path, subscription_id, resource_group_name, account_name, management_policy_name
+    );
+    let mut req_builder = client.delete(uri_str);
+    if let Some(token) = &configuration.bearer_access_token {
+        req_builder = req_builder.bearer_auth(token);
+    }
+    req_builder = req_builder.query(&[("api-version", &configuration.api_version)]);
+    let req = req_builder.build()?;
+    let res = client.execute(req).await?;
+    Ok(res.json().await?)
+}
+pub async fn private_endpoint_connections_list(
+    configuration: &Configuration,
+    resource_group_name: &str,
+    account_name: &str,
+    subscription_id: &str,
+) -> Result<PrivateEndpointConnectionListResult> {
+    let client = &configuration.client;
+    let uri_str = &format!(
+        "{}/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Storage/storageAccounts/{}/privateEndpointConnections",
+        &configuration.base_path, subscription_id, resource_group_name, account_name
+    );
+    let mut req_builder = client.get(uri_str);
+    if let Some(token) = &configuration.bearer_access_token {
+        req_builder = req_builder.bearer_auth(token);
+    }
+    req_builder = req_builder.query(&[("api-version", &configuration.api_version)]);
+    let req = req_builder.build()?;
+    let res = client.execute(req).await?;
+    Ok(res.json().await?)
+}
+pub async fn private_endpoint_connections_get(
+    configuration: &Configuration,
+    resource_group_name: &str,
+    account_name: &str,
+    subscription_id: &str,
+    private_endpoint_connection_name: &str,
+) -> Result<PrivateEndpointConnection> {
+    let client = &configuration.client;
+    let uri_str = &format!(
+        "{}/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Storage/storageAccounts/{}/privateEndpointConnections/{}",
+        &configuration.base_path, subscription_id, resource_group_name, account_name, private_endpoint_connection_name
+    );
+    let mut req_builder = client.get(uri_str);
+    if let Some(token) = &configuration.bearer_access_token {
+        req_builder = req_builder.bearer_auth(token);
+    }
+    req_builder = req_builder.query(&[("api-version", &configuration.api_version)]);
+    let req = req_builder.build()?;
+    let res = client.execute(req).await?;
+    Ok(res.json().await?)
+}
+pub async fn private_endpoint_connections_put(
```

Comment:
We should discuss name spacing. Having all of these as top level functions seems messy, but maybe it's the simplest and therefore best option.